### PR TITLE
snowflake 0.13.0

### PR DESCRIPTION
**Pull request by msentissi**
snowflake 0.13.0 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-5868](https://anaconda.atlassian.net/browse/PKG-5868) 
- [Upstream repository](https://pypi.org/project/snowflake)

### Explanation of changes:

- Updated to v0.13.0

[PKG-5868]: https://anaconda.atlassian.net/browse/PKG-5868?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by lorepirri** [comment]
pypi inspector: https://inspector.pypi.io/project/snowflake/0.13.0

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 383aee9dc305f6fe8fa04734fa778ecfa220da116de6a713edc30cacdc19a171
+  sha256: c47157446949ef34afc884fa6d42488e450cf3c0336ef12bed7e47f61d38ade5
   patches:
     # update 0.12.1: patch still needed
```
Comment (lorepirri) [inline recipe/meta.yaml:12]:
```suggestion
    # update 0.13.0: patch still needed
```

**Review by lorepirri — APPROVED** [review]
LGTM,
I checked if the patch is still needed. It is.

File: recipe/meta.yaml (hunk 1 above)
Comment (msentissi) [reply recipe/meta.yaml:12]: I had checked but forgot to update the comment, thank you !

**Review by Cansisti — APPROVED** [review]

**Pull request merged**